charts/tidb-cluster/templates/_helpers.tpl

```diff
@@ -14,3 +14,11 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{- define "tidb-cluster.utils.template" -}}
```

Comment:
I would like to give the template helpers generic names if they are not specific to our chart. This makes it easier to re-use them in different charts.

Reply:
Maybe `chart.configmap-template`?

Reply:
Maybe use `helm-toolkit.utils.template` instead @gregwebs @tennix ?

Reply:
ok, lgtm.